src/modal/index.tsx

```diff
@@ -41,7 +41,7 @@ function ModalWithAnalyticsFunnel({
       funnelType="modal"
       optionalStepNumbers={[]}
       totalFunnelSteps={1}
-      funnelNameSelectors={[`[${DATA_ATTR_MODAL_ID}="${CSS.escape(modalId)}"] .${styles['header--text']}`]}
+      funnelNameSelectors={[`[${DATA_ATTR_MODAL_ID}="${CSS?.escape(modalId)}"] .${styles['header--text']}`]}
```

Comment:
There is a bug. When CSS.escape is undefined, you will get `[${DATA_ATTR_MODAL_ID}="undefined"]`

Reply:
Also, in native JSDOM, the entire `CSS` object is undefined. A proper check would be `window?.CSS?.escape`